### PR TITLE
TC_00.001.18 | Creating new item > Creating Freestyle project

### PR DESCRIPTION
**Pull request by VitalyRezanov**
Add test for create creating Freestyle project

### CODE REVIEW

**Comment by VitalyRezanov** [comment]
[TC_00.001.18 | Creating new item > Creating Freestyle project](https://trello.com/c/l778yulL/538-tc0000118-creating-new-item-creating-freestyle-project)

File: src/test/java/school/redrover/FreestyleProject19Test.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+import java.util.List;
+
+public class FreestyleProject19Test extends BaseTest {
+    @Test
+    public void createProjectTest() {
```
[the remark this answers is not in the document]
Comment (VitalyRezanov) [reply src/test/java/school/redrover/FreestyleProject19Test.java]: I suppose, when viewing other similar tests, that the name of the method should start with "test". Did exactly that.

File: src/test/java/school/redrover/FreestyleProject19Test.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        final String projectName = "FreestyleProject19Test";
+        final WebDriver driver = getDriver();
+
+        //Click the New Item
```
[the remark this answers is not in the document]
Comment (VitalyRezanov) [reply src/test/java/school/redrover/FreestyleProject19Test.java]: Done

File: src/test/java/school/redrover/FreestyleProject19Test.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        driver.findElement(By.xpath("//*[@id=\"tasks\"]/div[1]/span/a")).click();
```
[the remark this answers is not in the document]
Comment (VitalyRezanov) [reply src/test/java/school/redrover/FreestyleProject19Test.java]: Done

File: src/test/java/school/redrover/FreestyleProject19Test.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        driver.findElement(By
+                .xpath("//*[@id=\"j-add-item-type-standalone-projects\"]/ul/li[1]"))
+                .click();
+
+        //Click the Ok button
+        driver.findElement(By.xpath("//*[@id=\"ok-button\"]")).click();
+
+        //Enter the description in the same name field
+        driver.findElement(By
+                        .xpath("//*[@id=\"main-panel\"]/form/div[1]/div[2]/div/div[2]/textarea"))
```
[the remark this answers is not in the document]
Comment (VitalyRezanov) [reply src/test/java/school/redrover/FreestyleProject19Test.java]: Done

**Review by Ksu0007 — APPROVED** [review]

**Review by KaterynaYazici — APPROVED** [review]

**Review by KaterynaYazici — APPROVED** [review]

**Comment by VitalyRezanov** [comment]
[TC_00.001.19 | Creating new item > Creating Freestyle project](https://trello.com/c/l778yulL/538-tc0000119-creating-new-item-creating-freestyle-project)

**Pull request merged**